pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -119,7 +120,46 @@ func shutdownAppsGlobal(ctx *zedagentContext) {
 	shutdownApps(ctx.getconfigCtx)
 }
 
-var baseosPrevConfigHash []byte
+var baseOSPrevConfigHash []byte
```

Comment:
You can't have the same hash used for two different parsed objects. Need one per.

Reply:
They are separated.

Reply:
That isn't what the diffs showed.

Reply:
Few lines below: https://github.com/lf-edge/eve/pull/2147/files#diff-6cd80a88d64c7180704bb38aea72a821d1214b3791a1b2fa0610f5a596e10ad1R162